modules/services/wikidata.js

```diff
@@ -32,9 +32,11 @@ export default {
             search: query,
             type: 'item',
             // the language to search
-            language: lang,
+            // do not include the country-code suffix
+            language: lang.split('-')[0],
             // the language for the label and description in the result
-            uselang: lang,
+            // do not include the country-code suffix
+            uselang: lang.split('-')[0],
```

Comment:
[This API query](https://www.wikidata.org/wiki/Special:ApiSandbox#action=query&format=json&meta=siteinfo&formatversion=2&siprop=languages) returns the list of interface languages supported by MediaWiki. However, MediaWiki is more lenient; you can use `en-NZ` as an interface language to get the expected results in English. (Using a more obscure interface language tends to prevent a site’s customized messages from coming through, but this is irrelevant for item queries.)